pkgs/development/interpreters/supercollider/default.nix

```
@@ -20,6 +20,10 @@ mkDerivation rec {
     ./supercollider-3.12.0-env-dirs.patch
   ];
 
+  postPatch = ''
+    sed -i "s,/bin/sh,${bash}/bin/sh," common/sc_popen.cpp
```

```suggestion
    substituteInPlace common/sc_popen.cpp --replace '/bin/sh' '${runtimeShell}'
```

Do you think you could `substituteInPlace common/sc_popen.cpp --replace '/bin/sh' '${runtimeShell}'` here instead?

It should be slightly more robust and doesn't need regex